source/trackers.json

```diff
@@ -879,6 +885,7 @@
         "switch.tv": "switchtv",
         "solaredge.com": "solaredge",
         "sonos.com": "sonos",
+        "streamotion.com.au": "streamotion",
```

Comment:
This company also has other streaming services: kayosports.com.au, binge.com.au

Reply:
oh, saw the next [pr](https://github.com/AdguardTeam/companiesdb/pull/204), can they be classified as the same company?

Reply:
Thanks for the detailed explanation. Option where could be added one Foxtel company and separate trackers for each service looks good to me.
@Alex-302 what do you think?

Reply:
I agree.